benefits/eligibility/forms.py

```diff
@@ -72,9 +72,7 @@ def add_api_errors(self, form_errors):
 
         if len(validation_errors) > 0:
             logger.warning("Form fields are invalid")
-
-        for (field, err) in validation_errors.items():
-            self.add_error(field, err)
+            self.add_error(None, validation_errors)
```

Comment:
Trying to simplify how errors are added to the form - to avoid sending extra params like `error.__all__` to Amplitude.